app/util/jmeter/start_jmeter_ui.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+import yaml
+from pathlib import Path
+from subprocess import run
+
+JIRA = "jira"
+CONFLUENCE = "confluence"
+BITBUCKET = "bitbucket"
+JSM = "jsm"
+APP_DIR = Path(__file__).resolve().parents[2]
+PROPERTIES = APP_DIR / "util" / "jmeter" / "jmeter.properties"
+JIRA_YML = APP_DIR / "jira.yml"
+CONFLUENCE_YML = APP_DIR / "confluence.yml"
+BITBUCKET_YML = APP_DIR / "bitbucket.yml"
+JSM_YML = APP_DIR / "jsm.yml"
+JIRA_JMX = APP_DIR / "jmeter" / "jira.jmx"
+CONFLUENCE_JMX = APP_DIR / "jmeter" / "confluence.jmx"
+BITBUCKET_JMX = APP_DIR / "jmeter" / "bitbucket.jmx"
+JSM_JMX = APP_DIR / "jmeter" / "jsm.jmx"
+JMETER_HOME = Path().home() / '.bzt' / 'jmeter-taurus'
+
+
+class StartJMeter:
+
+    def __init__(self):
+        self.env_settings = dict()
+        self.jmeter_properties = dict()
+        parser = argparse.ArgumentParser(description='Edit yml config')
+        parser.add_argument('--app', type=str, help='e.g. --app jira/confluence/bitbucket/jsm')
+        self.args = parser.parse_args()
+        if not self.args.app:
+            raise SystemExit('Application type is not specified. e.g. --app jira/confluence/bitbucket/jsm')
+        if self.args.app == JIRA:
+            self.yml = JIRA_YML
+            self.jmx = JIRA_JMX
+        elif self.args.app == CONFLUENCE:
+            self.yml = CONFLUENCE_YML
+            self.jmx = CONFLUENCE_JMX
+        elif self.args.app == BITBUCKET:
+            self.yml = BITBUCKET_YML
+            self.jmx = BITBUCKET_JMX
+        elif self.args.app == JSM:
+            self.yml = JSM_YML
+            self.jmx = JSM_JMX
+        else:
+            raise SystemExit(f"Application type {self.args.app} is not supported. "
+                             f"Valid values: {JIRA} {CONFLUENCE} {BITBUCKET} {JSM}")
+
+    @staticmethod
+    def read_yml_file(file):
+        with file.open(mode='r') as file:
+            return yaml.load(file, Loader=yaml.FullLoader)
+
+    @staticmethod
+    def update_properties_file(content):
+        with open(PROPERTIES, 'w') as f:
```

Comment:
How do you think, maybe "f" change on "file" like in the previous read_yml_file method

Reply:
good idea, done